src/components/record/RecordForm.tsx

```diff
@@ -170,6 +170,9 @@ export default function RecordForm(props: Props) {
             disabled={!allowEditing}
             record={JSON.stringify(recordData, null, 2)}
             onSubmit={handleOnSubmit}
+            attachmentEnabled={attachment?.enabled}
+            // require an attachment on initial insert if attachments are required
```

Comment:
This comment could mention the fact that we don't want to require attachments on existing records, because we want to be able to edit the fields without having to reupload its attachment? Or is it obvious?